test/index.js

```diff
@@ -82,24 +82,6 @@ describe('bbcode', function() {
 });
 
 describe('commands', function() {
-    describe('/captionbot', function() {
```

Comment:
Since mehdown is public, does removal of a command warrant a full version bump?

Reply:
For a command that is rarely used and that's already been broken for several months a minor version bump seems fine.